frontend/app/routes/protected/profile/edit-communication-preferences.tsx

```diff
@@ -79,19 +79,19 @@ export async function action({ context: { appContainer, session }, params, reque
     preferredMethodSunLife: z
       .string()
       .trim()
+      .toLowerCase()
       .min(1, t('protected-profile:edit-communication-preferences.error-message.preferred-method-sunlife-required'))
       .refine(
-        // TODO: check if email is verified once PP has updated the clientApplication payload to include that field
-        (val) => val !== COMMUNICATION_METHOD_SUNLIFE_EMAIL_ID || clientApplication.contactInformation.email !== undefined,
+        (val) => val !== COMMUNICATION_METHOD_SUNLIFE_EMAIL_ID || clientApplication.contactInformation.email !== undefined || clientApplication.contactInformation.emailVerified === true,
```

Comment:
It's `AND` condition.
```suggestion
        (val) => val !== COMMUNICATION_METHOD_SUNLIFE_EMAIL_ID || (clientApplication.contactInformation.email !== undefined && clientApplication.contactInformation.emailVerified),
```